packages/python-debian/python-debian.spec

```diff
@@ -67,6 +68,11 @@ set -ex
 
 
 %changelog
+* Mon Aug 22 2022 Quirin Pamp <pamp@atix.de> - 0.1.44-2
+- Update to 0.1.44
+- Add zstd dependency to support zstd compression for Ubuntu.
+- Use revision 2 to ensure build against python 3.8.
+
 * Wed Nov 03 2021 Odilon Sousa 0.1.42-1
```

Comment:
I did not cherry-pick any of the intermediary revisions to this file from the `rpm/3.18` branch. Is that Ok, or should I have done so?

Reply:
That's ok